requirements.txt

```diff
@@ -1,5 +1,7 @@
 checklist==0.0.10
 spacy==2.2.4
+nltk==3.6.2
```

Comment:
It would be good to put the requirements in the transformation folder like did in [this one](https://github.com/GEM-benchmark/NL-Augmenter/blob/48135d4c636b304a0c38d1cc4136db7e600c23b8/transformations/punctuation/requirements.txt). This is to avoid other people's transformation might change the dependency version and might lead to a conflict.

Reply:
Updated.